packages/salesforcedx-vscode-lightning/package.json

```diff
@@ -126,13 +126,28 @@
     ],
     "configuration": {
       "type": "object",
-      "title": "%feature_previews_title%",
+      "title": "%lightning_preferences%",
       "properties": {
         "salesforcedx-vscode-lightning.show-lightning-explorer": {
           "type": "boolean",
           "scope": "window",
           "default": false,
           "description": "%show_lightning_explorer_description%"
+        },
+        "salesforcedx-vscode-lightning.activationMode": {
+          "type": "string",
+          "description": "%activation_mode_description%",
+          "enum": [
+            "always",
+            "autodetect",
+            "off"
+          ],
+          "enumDescription": [
+            "%activation_mode_always_on%",
```

Comment:
Could `always` and `autodetect` be merged ?

Reply:
they are slightly different. Autodetect will not start the LSP if it doesn't find the correct workspace structure, whereas 'always' will always run the LSP regardless of workspace structure